src/scenes/PlantsDashboardRouter/components/PlantingSiteTrendsCard.tsx

```diff
@@ -0,0 +1,130 @@
+import { useMemo } from 'react';
+
+import React, { Box, Typography, useTheme } from '@mui/material';
+import { Icon, Tooltip } from '@terraware/web-components';
+
+import Card from 'src/components/common/Card';
+import Chart from 'src/components/common/Chart/Chart';
+import { selectObservationsResults } from 'src/redux/features/observations/observationsSelectors';
+import { selectPlantingSite } from 'src/redux/features/tracking/trackingSelectors';
+import { useAppSelector } from 'src/redux/store';
+import strings from 'src/strings';
+
+type PlantingSiteTrendsCardProps = {
+  plantingSiteId: number;
+};
+
+export default function PlantingSiteTrendsCard({ plantingSiteId }: PlantingSiteTrendsCardProps): JSX.Element {
+  const theme = useTheme();
+  const allObservationsResults = useAppSelector(selectObservationsResults);
+  const plantingSite = useAppSelector((state) => selectPlantingSite(state, plantingSiteId));
+
+  const totalArea = plantingSite?.areaHa ?? 1;
+
+  const siteObservations = useMemo(() => {
+    if (!allObservationsResults || !plantingSiteId) {
+      return [];
+    }
+    return allObservationsResults?.filter((observationResult) => {
+      const matchesSite = observationResult.plantingSiteId === plantingSiteId;
+      return matchesSite;
+    });
+  }, [allObservationsResults, plantingSiteId]);
+
+  const sortedObservations = useMemo(
+    () => siteObservations.sort((a, b) => new Date(a.startDate).getTime() - new Date(b.startDate).getTime()),
+    [siteObservations]
+  );
+
+  const separatorStyles = {
+    width: '1px',
+    height: 'auto',
+    backgroundColor: theme.palette.TwClrBrdrTertiary,
+    marginRight: '24px',
+    marginLeft: '24px',
+  };
+
+  const plantsChartData = useMemo(() => {
+    const labels = sortedObservations.map((ob) => ob.startDate);
+    const values = sortedObservations.map(
```

Comment:
@sgrimm do you know if this is the correct way of calculating plants per ha? Or should I be using other value?
I'm summing up all total plants from all the zones and diving it by the totalArea of the planting site

Reply:
The server includes `estimatedPlants` fields in the site, zone, and subzone observation results payloads, which takes the areas into account already. But that only counts subzones that have been marked "planting completed" which maybe isn't what we want here.

I'm having a hard time finding any mention of a plants-per-hectare graph in the PRD; can you point me to it? I can't give you a solid answer because I'm not actually sure what this value is supposed to mean, but hopefully the PRD has more detail.

Reply:
If I had to guess at what the intent of that graph was, I'd say the right calculation here is maybe to total up all the live (and existing, maybe? not sure) plants across all species at the _plot_ level for the observation, then divide that by the total hectares of all the plots.

I think you don't want to use the total hectares of the site or even the zone because the observation might only be covering a small user-selected part of the site.

Calculating the value the way I just described will probably result in a graph that fluctuates wildly from observation to observation if people are observing small areas with different mixes of species but in the absence of any definition to the contrary, that's what I'd go with.

Reply:
@alexTerraformation, does that seem right to you?

Reply:
I didn't find it in the PRD, but this is the design: https://www.figma.com/design/MoIzWBItJSC4t8UDSCkG95/Changeable-Plant-Monitoring?node-id=1665-32320&node-type=frame&t=JOfdniO8FDwG5Q1g-0

Reply:
Also commented on the design, but I guess I'll take my questions to the Jira task for further discussion.